Yale mods

jambun opened the pull request:
Hi,

We're making a few modifications for Yale.

Sending this to let you know this is happening and see if you would be interested in merging these changes at some stage. They are currently being tested.

We expect to be making a few more changes as part of this project.


Cheers,
James

AustinTSchaffer commented on public/lib/record_mapper.rb:
```diff
@@ -14,6 +14,20 @@ def repo_settings
+    # If #show_action? returns false, then the button is shown disabled
+    # This method tests whether the button should be hidden (rather than just disabled)
+    # This is controlled by two repo config options, which default to false to maintain
+    # the existing functionality.
```
The first line of this comment seems irrelevant to the `hide_button?` method, and could be moved to above the `show_action?` method. We believe that the following wording would be more appropriate for this method's documentation.

```
This method tests whether the button should be hidden. This determination is based 
on the settings for the repository and defaults to false.
```

AustinTSchaffer commented on public/views/aeon/_aeon_request_action.html.erb:
```diff
@@ -14,8 +14,10 @@ mapper = case record
 
+<% unless mapper.hide_button? %>
+
 <% if mapper.show_action? %>
```
Please indent the contents of the `unless mapper.hide_button?` block by 2 spaces.

jambun commented:
Hi @AustinTSchaffer,

Just pushed a few commits addressing your points.

My goal originally was to touch as little code as possible - happy to make these changes now.


Cheers,
James
